### DISCUSSION TITLE
Conecting Gopro hero 7 to phone app with non working screen

### DISCUSSION THREAD

**Discussion by Sixilef** [gopro/labs#356] [Q&A] [answered]
Hello, im looking for a way to connect my gopro hero 7 to the phone app with not working display (the mini display close to camera lens is working) so i can fully control the gopro. it seems impossible before i discovered the gopro labs firmware but now i think its possible with Math QR Codes
im wondering if someone know how to create a code that can connect my gopro to the wifi (a code that i write wifi credentials and connect it automatically)
If someone could help i would really appreciate it 🙏
thanks!

**Comment by dnewman-gpsw** [comment, marked as answer]
The command !PA will initiate app pairing.  You will need to press the shutter three times (quickly) after scanning that code, to approve the pairing on camera.

**Reply by javiiirs07** [reply]
and for Gopro hero 11 what code I can use to connect the app again with a new phone? Thank you.

**Reply by dnewman-gpsw** [reply]
same code.

**Reply by Sixilef** [reply]
Its not working, my gopro doesn't go on pairing mode when i scan the !PA QR Code (im using gopro official app and it says no gopro detected)
https://gopro.github.io/labs/control/set/?cmd=!PA
when i try other commands like changing resolution and framerate its working but this is not working (i tryed also !FORMAT and its not working)

**Reply by dnewman-gpsw** [reply]
!FORMAT is not offered on HERO7, that was added later to Labs.  

With !PA did you press the shutter three times quickly?  I have confirmed it works on HERO7.

**Reply by Sixilef** [reply]
Oh, it worked thanks you so much dude that saved me a lot of time
i really appreciate your help 🙏

**Reply by attila0307** [reply]
Dear David,

Is it working with MAX?

**Reply by dnewman-gpsw** [reply]
Yes.

**Reply by Arhrocket** [reply]
Thanks for your advices, it's work! Gopro 7 black. This method works. (Gopro with broken led and forget wi-fi password). 1) Download lab firmware (LABS_HERO7_01_90_71.zip) 2) Extract update folder to micro sd ( to the root folder) 3) insert micros sd and turn on gopro, wait 2 minutes ( will auto flash in 2 steps, after finished beeps) 4) press mode to change mode to photo 5) Scan !PA code (https://gopro.github.io/labs/control/set/?cmd=!PA) and press fastly 3 times record button 6) Open app android phone GoPro Quik 11.20 , resume as quest, tap connect go pro. Profit! Connected and you could change all settings.
